<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateGroupPage</class>
 <widget class="QDialog" name="CreateGroupPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Creat Group</string>
  </property>
  <widget class="QLabel" name="groupname_lbl">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Segoe Script&quot;;</string>
   </property>
   <property name="text">
    <string>Group Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="groupname_led">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="result_lbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Group result...</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>100</width>
     <height>41</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="creategroup_pbn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Segoe Script&quot;;</string>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="joingroup_pbn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Segoe Script&quot;;</string>
     </property>
     <property name="text">
      <string>Join</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolButton" name="tojoin_tbn">
   <property name="geometry">
    <rect>
     <x>209</x>
     <y>200</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>31</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="tocreate_tbn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>31</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_pbn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Segoe Script&quot;;</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QToolButton" name="Exit_pbn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
